assimp/importer.hpp: No such file or directory

```
In file included from /home/roman/dev/opengl/Glitter/Glitter/Sources/main.cpp:2:0:
/home/roman/dev/opengl/Glitter/Glitter/Headers/glitter.hpp:5:31: fatal error: assimp/importer.hpp: No such file or directory
 #include <assimp/importer.hpp>
                               ^
compilation terminated.
make[2]: *** [CMakeFiles/Glitter.dir/Glitter/Sources/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Glitter.dir/all] Error 2
make: *** [all] Error 2
```

--- a/Glitter/Headers/glitter.hpp
+++ b/Glitter/Headers/glitter.hpp
@@ -2,7 +2,7 @@
 #define STB_IMAGE_IMPLEMENTATION
 
 // System Headers
-#include <assimp/importer.hpp>
+#include <assimp/Importer.hpp>
 #include <assimp/postprocess.h>
 #include <assimp/scene.h>
 #include <glad/glad.h>

--- a/Glitter/Sources/main.cpp
+++ b/Glitter/Sources/main.cpp
@@ -3,7 +3,7 @@
 
 // System Headers
 #include <glad/glad.h>
-#include <glfw/glfw3.h>
+#include <GLFW/glfw3.h>
 
 // Standard Headers
 #include <cstdio>

